scripts/rotate-aws-creds.sh

```diff
@@ -1,216 +0,0 @@
-#!/bin/bash
-
-ENVIRONMENT=$1
-CLUSTER_ID=$2
-
-# make sure we have no state for the aws cli
-unset AWS_PROFILE
-unset AWS_ACCESS_KEY_ID
-unset AWS_SECRET_ACCESS_KEY
-# Region is a mandatory value to be set when using AWS CLI.
-# All AWS CLI requests are related to IAM and are global, so picking any region will be OK.
-export AWS_DEFAULT_REGION=us-east-1
-
-if [ "$CLUSTER_ID" == "" ] || [ "$ENVIRONMENT" == "" ];
-then
-    echo "usage: $0 <environment> <cluster id>"
-    echo "example: $0 stage 18kshg9idnn687p1bstlh669hlod7ldr"
-    exit 1
-fi
-
-info() {
```

Comment:
This function is called frequently in the script, but no longer exists, so it executes the builtin `info` command rather than printing a log message.

Reply:
Fixed